components/collection/activity/events/eventRow/EventRowDesktop.vue

```diff
@@ -0,0 +1,144 @@
+            {{ event.nft.name }}
+          </div>
+        </nuxt-link>
+      </div>
+    </div>
+
+    <div class="column">
+      <div class="height-50px is-flex is-align-items-center">
+        <div
+          class="border is-size-7 is-justify-content-center py-1 my-2 is-flex is-align-items-center fixed-width fixed-height"
+          :class="getInteractionColor(event.interaction)">
+          {{ interactionName }}
+        </div>
+      </div>
+    </div>
+
+    <div class="column">
+      <div class="height-50px is-flex is-align-items-center">
+        <div v-if="amount === blank">
+          {{ blank }}
+        </div>
+        <Money v-else :value="amount" />
+      </div>
+    </div>
+
+    <div class="column">
+      <div class="height-50px is-flex is-align-items-center">
+        <nuxt-link
+          v-if="fromAddress !== blank"
+          :to="`/${urlPrefix}/u/${fromAddress}`"
+          class="has-text-link">
+          <IdentityIndex ref="identity" :address="fromAddress" show-clipboard />
+        </nuxt-link>
+        <div v-else>
+          {{ blank }}
+        </div>
+      </div>
+    </div>
+    <div class="column">
+      <div class="height-50px is-flex is-align-items-center">
+        <nuxt-link
+          v-if="toAddress !== blank"
+          :to="`/${urlPrefix}/u/${toAddress}`"
+          class="has-text-link">
+          <IdentityIndex ref="identity" :address="toAddress" show-clipboard />
+        </nuxt-link>
+        <div v-else>
+          {{ blank }}
+        </div>
+      </div>
+    </div>
+    <div class="column">
+      <div class="height-50px is-flex is-align-items-center">
+        {{ timeAgo(event.timestamp) }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { InteractionWithNFT, Offer } from '@/components/collection/utils/types'
+import Money from '@/components/shared/format/ChainMoney.vue'
+import IdentityIndex from '@/components/identity/IdentityIndex.vue'
+import { timeAgo } from '@/components/collection/utils/timeAgo'
+import {
+  blank,
+  getAmount,
+  getFromAddress,
+  getInteractionColor,
+  getNFTAvatar,
+  getToAddress,
+  interactionNameMap,
+} from './common'
+
+const { urlPrefix } = usePrefix()
+const props = defineProps<{
+  event: InteractionWithNFT | Offer
+}>()
+
+const avatar = ref<string>()
+
+const interactionName = computed(
+  () => interactionNameMap[props.event.interaction] || props.event.interaction
+)
+const amount = computed(() => getAmount(props.event))
+
+const fromAddress = computed(() => getFromAddress(props.event))
+
+const toAddress = computed(() => getToAddress(props.event))
+
+onMounted(() => {
+  getAvatar()
+})
+
+const getAvatar = async () => {
+  if (props.event) {
+    avatar.value = await getNFTAvatar(props.event)
+  }
+}
+</script>
+
+<style scoped lang="scss">
+.fixed-width {
+  width: 66px;
+}
+.fixed-height {
+  height: 22px;
+}
+.height-50px {
+  height: 50px;
+}
```

Comment:
do we need to set fixed height & width?

Reply:
long discussion about this with @exezbcz  in discord...
yes

Reply:
altough fixed height can probably be dropped

Reply:
is that for the tags? :D

Reply:
dropping fixed height causes height diff between desktop and tablet view
i guess min-height could solve it, but what would be the point of that then?